ios/Example/Tests/CertificateTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CertificateTests.swift
+//  Trifle_Tests
+//
+//  Created by Gelareh Taban on 5/17/23.
```

Comment:
Let's remove author and copyright headers

Reply:
sounds good